external/mqtt/pom.xml

```diff
@@ -72,7 +72,6 @@
       <groupId>org.apache.activemq</groupId>
       <artifactId>activemq-core</artifactId>
       <version>5.7.0</version>
-      <scope>test</scope>
```

Comment:
Didn't realize we need to bring this one to the compile scope for tests. I feel it may be not worth adding it only for tests.
 
@tdas what do you think about this dependency?